src/components/EpisodeCard.tsx

```diff
@@ -0,0 +1,38 @@
+import IEpisode from "../interfaces/IEpisode";
+import cleanString from "../util/cleanString";
+import padNumber from "../util/padNumber";
+import "./episodeCard.css";
+
+interface EpisodeCardProps {
+    key: number;
+    episode: IEpisode;
+}
+
+export function EpisodeCard(props: EpisodeCardProps): JSX.Element {
+    const e = props.episode;
+    const paddedSeasonNumber = padNumber(e.season);
+    const paddedEpisodeNumber = padNumber(e.number);
+    let cleanedSummary = cleanString(e.summary, "<p>");
+    cleanedSummary = cleanString(cleanedSummary, "</p>");
+    cleanedSummary = cleanString(cleanedSummary, "<br>");
+    cleanedSummary = cleanString(cleanedSummary, "<br />");
+    cleanedSummary = cleanString(cleanedSummary, "<b>");
+    cleanedSummary = cleanString(cleanedSummary, "</b>");
+    cleanedSummary = cleanString(cleanedSummary, "</i>");
+    cleanedSummary = cleanString(cleanedSummary, "<i>");
+
+    return (
+        <div className="card-container">
+            <div className="episode-heading">
+                {e.name}{" "}
+                <div className="episode-number">
+                    S{paddedSeasonNumber}E{paddedEpisodeNumber}
```

Comment:
Could be a seperate function to combine and return everything together. Not easy to read at the moment